UI/bower_BACKUP_4748.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
@MHudak Looks like there are some bower jsons that need to be taken out from the PR.